modules/app/webhook.tf

```diff
@@ -0,0 +1,57 @@
+// Generate a random webhook secret
+resource "random_password" "webhook-secret" {
```

Comment:
Is something supposed to pass this secret to GitHub somehow? Or is this supposed to be an input instead of a randomly generated internal thing? I don't understand how app installation webhooks are secured.

Reply:
Yeah, basically we need to access the secret and feed it into github when we register the webhook portion of things.

Since the webhook reads this at startup I went this route mostly for bootstrapping purposes.  I really wish github would switch to a model more like ours.